test/html.test.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const assert = require('assert');
+const jsdom = require('jsdom').jsdom;
+const html = require('../lib/html');
+
+describe('html', function() {
+  before(function() {
+    global.document = jsdom();
+  });
+
+  after(function() {
+    delete global.document;
+  });
+
+  describe('raw', function() {
+    it('should export a method', function() {
+      assert.equal(typeof html.raw, 'function');
+    });
+
+    it('should create a new Markup instance', function() {
+      const inst = html.raw();
+      assert.equal(inst.constructor.name, 'Markup');
+    });
+
+    describe('Markup instance', function() {
+      it('should store an internal value', function() {
+        const inst = html.raw('foo');
+        assert.equal(inst.value, 'foo');
+      });
+
+      it('should implement a toString method and output value', function() {
+        const inst = html.raw('foo');
+        assert.equal(inst.toString(), 'foo');
+      });
+    });
+  });
+
+  describe('isRaw', function() {
+    it('should export a method', function() {
+      assert.equal(typeof html.isRaw, 'function');
+    });
+
+    it('should identify raw Markup instances', function() {
+      const inst = html.raw('foo');
+      assert.equal(html.isRaw(inst), true);
+    });
+
+    it('should return false for other objects', function() {
+      const obj = { value: 'foo' };
+      assert.equal(html.isRaw(obj), false);
+    });
+  });
+
+  describe('onerror', function() {
+    it('should export null by default', function() {
+      assert.equal(html.onerror, null);
```

Comment:
again, I'd use a `strictEqual` here checking against `null` because this would accept `undefined`. I _do_ try to use `assert.equal` over `assert.strictEqual` just because it's easier to type and I think it reads better.